packages/clarity-js/src/data/upload.ts

```diff
@@ -77,8 +77,9 @@ export function queue(data: Token[]): void {
                 } else { metric.count(Metric.LayoutBytes, event.length); }
                 break;
             case Event.BoxModel:
+                metric.count(Metric.RegionBytes, event.length);
+                break;
             case Event.Document:
-                metric.count(Metric.LayoutBytes, event.length);
```

Comment:
Please add a comment why this case doesn't count for anything. It's less obvious than some of the others.

Reply:
Done